user-guide/Reference/DataMiner_Tools/Standalone_Elasticsearch_Cluster_installer/Standalone_Elasticsearch_Cluster_Installer.md

```diff
@@ -101,25 +99,24 @@ After installation, Elasticsearch will by default only use 4 GB of heap space. I
 ### Using the Registry Editor
 
 1. Go to `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Apache Software Foundation\Procrun 2.0\elasticsearch-service-x64\Parameters\Java`
-
-1. Open the *Options* key and locate the `-xmsZg` and `-xmxZg` settings. In both settings, change “Z” to the amount of heap space (in gigabytes) you want Elasticsearch to use (example: -xms16 and -xmx16g). Default: 1
-
-1. Restart the Elasticsearch service.
+2. Open the *Options* key and locate the `-xmsZg` and `-xmxZg` settings. In both settings, change “Z” to the amount of heap space (in gigabytes) you want Elasticsearch to use (example: -xms16 and -xmx16g). Default: 1
+3. Restart the Elasticsearch service.
 
 ### Using the Elasticsearch service manager
 
 1. Open a command window and go to the Elasticsearch installation folder. Default: `C:\Program Files\Elasticsearch`.
-
-1. Go to the bin folder, and run `elasticsearch-service.bat manager`.
-
-1. In the service manager window, go to the *Java* tab, and locate the `-xmsZg` and `-xmxZg` settings. In both settings, change “Z” to the amount of heap space (in gigabytes) you want Elasticsearch to use (example: -xms16 and -xmx16g). Default: 1
-
-1. Click *Apply* and then click *OK*.
-
-1. Restart the Elasticsearch service.
+2. Go to the bin folder, and run `elasticsearch-service.bat manager`.
+3. In the service manager window, go to the *Java* tab, and locate the `-xmsZg` and `-xmxZg` settings. In both settings, change “Z” to the amount of heap space (in gigabytes) you want Elasticsearch to use (example: -xms16 and -xmx16g). Default: 1
+4. Click *Apply* and then click *OK*.
+5. Restart the Elasticsearch service.
```

Comment:
To create a numbered list, we always add "1. " at the start of each list item. When the documentation is generated, the numbering will automatically be adjusted to a correctly numbered list. While it is possible to use "1. ", "2. ", "3. ", etc., we do not recommend this, as it can lead to a lot of needless editing if something is added or removed in an existing numbered list.